src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,7 +1,56 @@
 import React from 'react';
-
 import './MovieCard.scss';
+import { Movie } from '../../types/Movie';
+
+type Props = {
+  movie: Movie
+};
+
+export const MovieCards: React.FC<Props> = ({ movie }) => {
+  const {
+    title,
+    description,
+    imgUrl,
+    imdbUrl,
+  } = movie;
+
+  return (
```

Comment:
The 'alt' attribute in img tag should have a more descriptive value, like 'Movie poster' or '{title} poster'.

Reply:
agree